docs/apps/index.md

```diff
@@ -113,6 +113,7 @@
 
 ## Geosciences
 
+* [Ames Stereo Pipeline](ames-stereo.md) for processing stereo images captured from satellites (around Earth and other planets).
```

Comment:
```suggestion
* [Ames Stereo Pipeline](ames-stereo.md) for processing stereo images
```
Since it is meant for all kinds of stereo image sources, I'd leave satellites out from here.

Reply:
This is direct copy-paste from their docs, so actually do not know, if it would work with aerial stereo data..

Reply:
In the first paragraph robotic rovers and aerial cameras are mentioned?